terraform/workspace_variables/production.tfvars.json

```diff
@@ -8,5 +8,22 @@
     "storage_log_categories": [],
     "resource_prefix": "s165p01-",
     "app_service_plan_sku": "P1v2",
-    "worker_count": 3
+    "worker_count": 3,
+    "statuscake_alerts": {
+      "get-an-identity-prod": {
+        "website_name": "get-an-identity-prod",
+        "website_url": "https://s165p01-getanid-production-auths-app.azurewebsites.net/status",
+        "test_type": "HTTP",
```

Comment:
This will be the same for any alert. Can we hardcode it in statuscake.tf instead so the config here is smaller?
Similarly for check_rate, trigger_rate, node_locations, etc

Reply:
done.